src/main/scala/chisel3/stage/ChiselAnnotations.scala

```diff
@@ -362,28 +362,37 @@ object ChiselOutputFileAnnotation extends HasShellOptions {
   */
 case class DesignAnnotation[DUT <: RawModule](design: DUT) extends NoTargetAnnotation with Unserializable
 
-/** Use legacy Chisel shift-right behavior
+/** Use legacy Chisel width behavior.
   *
   * '''This should only be used for checking for unexpected semantic changes when bumping to Chisel 7.0.0'''
   *
-  * Use as CLI option `--use-legacy-shift-right-width`.
+  * Use as CLI option `--use-legacy-width`.
   *
-  * This behavior is inconsistent between Chisel and FIRRTL
-  * - Chisel will report the width of a UInt or SInt shifted right by a number >= its width as a 0-bit value
-  * - FIRRTL will implement the width for these UInts and SInts as 1-bit
+  * There are two width bugs fixed in Chisel 7.0 that could affect the semantics of user code.
+  *
+  * 1. The width of shift-right when shift amount is >= the width of the argument
+  *
+  * This behavior is inconsistent between Chisel and FIRRTL.
+  * - Chisel will report the width of a UInt or SInt shifted right by a number >= its width as a 0-bit value.
```

Comment:
I'm a bit confused about the tense in this doc. So what is the state of affaris after this PR is merged?

Reply:
This part of doc is the same as before, but this is a good opportunity to tighten this phrasing up (especially since this should probably be turned into a doc on the website).

The present tense here is describing what enabling the flag does.

Reply:
I redid the ScalaDoc a little, hopefully it's more clear now.